src/components/result-view-tab.js

```diff
@@ -5,22 +5,32 @@ import Paper from '@mui/material/Paper';
 import LoadFlowResult from './loadflow-result';
 import makeStyles from '@mui/styles/makeStyles';
 import { useIntl } from 'react-intl';
-import { useState } from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { SecurityAnalysisResultTab } from './security-analysis-result-tab';
+import Alert from '@mui/material/Alert';
+import { allowModificationsOnNode } from './graph/util/model-functions';
 
 const useStyles = makeStyles((theme) => ({
+    div: {
+        display: 'flex',
+    },
     table: {
         display: 'flex',
         flexDirection: 'column',
         flexGrow: 1,
     },
+    invalidNode: {
+        marginTop: '4px',
+        marginLeft: '50px',
```

Comment:
Use theme.spacing(X) instead of pixels ?
theme.spacing(0.5) = 4px;
theme.spacing(6.25) = 50px;

Reply:
Done